collectors/HostSystemCollector.py

```diff
@@ -0,0 +1,50 @@
+from BaseCollector import BaseCollector
+import os, time, json
+from pathlib import Path
+from prometheus_client.core import GaugeMetricFamily
+from tools.Resources import Resources
+
+
+class HostSystemCollector(BaseCollector, Resources):
```

Comment:
because of line 40. I have written the function get_metric into the resource class.

Reply:
Please remove it from here. For L40 it is not needed, because it is explicitly called. I'd rather call Resources.get_hosts() instead of self, then this can be removed.